river/base/drift_detector.py

```diff
@@ -10,47 +9,32 @@ class DriftDetector(base.Base):
 
     def __init__(self):
         super().__init__()
-        self._in_concept_change = False
-        self._in_warning_zone = False
+        self._drift_detected = False
 
     def reset(self):
```

Comment:
I would make this method private

Reply:
Done!

Commit [063ca61](https://github.com/online-ml/river/pull/963/commits/063ca61d06f5335c05cf78234dc9549e70fc7a72) also ensures that every non-random detector calls the `_reset` method internally after a drift is detected.